walk-in-interview/src/main/java/com/google/job/data/Job.java

```diff
@@ -1,18 +1,99 @@
 package com.google.job.data;
 
+import jdk.internal.jline.internal.Nullable;
+
+import java.util.List;
+import java.util.Optional;
+
 /** Class for a job post. */
 public final class Job {
-    // TODO(issue/9): Add the feature to create a new job post
+    private static final String DUMMY = "dummy";
+
+    private String jobId;
```

Comment:
Can all these attributes be final? Or are they not final because of firestore?

Reply:
when retrieved the `Job` post from the cloud firestore and to be displayed on the web page, `jobId` will be reassigned with the id created by cloud firestore; just like what we did in the portfolio project

Reply:
Maybe here we can make everything final. When you got the jobId, create another instance with the newly available information? Be builder pattern Yohan recommends below is very useful for that as well.

Reply:
I fixed with builder pattern, but since that the variable will be reassigned when calling those setter for builder, I cannot make it final